sat/cli/bootsys/stages.py

```diff
@@ -72,5 +72,17 @@ def load_stage(bootsys_module, function):
         ('k8s-check', ('k8s', 'do_k8s_check')),
         ('cabinet-power', ('cabinet_power', 'do_cabinets_power_on')),
         ('bos-operations', ('bos', 'do_bos_boots'))
-    ])
+    ]),
+    'shutdown': OrderedDict([  # TODO: Ask Jason if right
+        ('capture-state', ('state_recorder', 'do_state_capture')),
+        ('session-checks', ('service_activity', 'do_service_activity_check')),
+        ('bos-operations', ('bos', 'do_bos_reboots')),
+        ('cabinet-power', ('cabinet_power', 'do_cabinets_power_off')),
+        ('platform-services', ('platform', 'do_platform_stop')),
+        ('ncn-power', ('mgmt_power', 'do_power_off_ncns')),
+        ('ncn-power', ('mgmt_power', 'do_power_on_ncns')),
+        ('platform-services', ('platform', 'do_platform_start')),
+        ('k8s-check', ('k8s', 'do_k8s_check')),
+        ('cabinet-power', ('cabinet_power', 'do_cabinets_power_on')),
+    ]),
```

Comment:
I think this is what you want here (untested):

```suggestion
    'reboot': OrderedDict([
        ('bos-operations', ('bos', 'do_bos_reboots')),
    ]),
```